src/dashboard/Data/Browser/AttachRowsDialog.react.js

```diff
@@ -28,13 +28,10 @@ export default class AttachRowsDialog extends React.Component {
       return [...resourceIds, objectId];
     }, []);
     const promise = new Parse.Promise();
```

Comment:
You can shorten this whole thing to just `return this.props.onConfirm(objectIds);` I think,